guides/en/2_advanced/8_particles.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Particles
+description: Learn how to create particles in KAPLAY.
+url: particles
+version: v4000
```

Comment:
Particle API is available in both versions, so is not necessary the version specifier